lib/validic/version.rb

```diff
@@ -1,3 +1,3 @@
 module Validic
-  VERSION = '0.4.1'
+  VERSION = '0.4.7'
```

Comment:
this should probably be a minor version bump since we are adding new functionality instead of a bugfix bump. also why the bump from .1 to .7?

I would suggest `0.5.0` http://semver.org/